lib/miq_environment.rb

```diff
@@ -9,10 +9,7 @@ def self.arch
   # Return the fully qualified hostname for the local host.
   #
   def self.fully_qualified_domain_name
-    hostname    = Socket.gethostname
-    addrinfo    = Addrinfo.getaddrinfo(hostname, nil, nil, :STREAM).first
-    fqdn, _port = addrinfo.getnameinfo
-    fqdn
+    @fully_qualified_domain_name ||= `hostname --fqdn`.chomp
```

Comment:
`--fqdn` doesn't work on a Mac, so this will break macOS devs

Reply:
Ok, yeah so if we can switch to `-f` that should just work

Reply:
> -f    Include domain information in the printed name.  This is the default behavior.

Oh wow so the default behavior is also different?  That's wild